nifi-nar-bundles/nifi-standard-services/nifi-hbase_2-client-service-bundle/nifi-hbase_2-client-service/src/main/java/org/apache/nifi/hbase/HBase_2_ClientService.java

```diff
@@ -413,8 +448,20 @@ public void shutdown() {
         if (connection != null) {
             try {
                 connection.close();
-            } catch (final IOException ioe) {
-                getLogger().warn("Failed to close connection to HBase due to {}", new Object[]{ioe});
+            } catch (final Exception e) {
+                getLogger().warn("Failed to close connection to HBase due to {}", new Object[]{e});
+            }
+        }
+
+        final KerberosUser kerberosUser = kerberosUserReference.get();
+        if (kerberosUser != null) {
+            try {
+                kerberosUser.logout();
+            } catch (final Exception e) {
+                getLogger().warn("Error logging out KerberosUser: {}", e.getMessage(), e);
```

Comment:
Is it necessary to include the exception message since it will also be included in the stack trace?
```suggestion
                getLogger().warn("KeberosUser Logout Failed", e);
```